### PR TITLE
Wallet Check-in

### PR DESCRIPTION
**Pull request by Parsh**
Includes: Last seen for TCs, Health fetch & update, Ex-rates

### CODE REVIEW

File: src/store/reducers/accounts.ts [hunk 1]
```diff
@@ -465,8 +465,8 @@ export default (state = initialState, action) => {
             settedup: false,
             loading: true,
           },
-        }
-      }
+        },
+      };
 
     case SETTED_DONATION_ACC:
```
Comment (AliMeer) [inline src/store/reducers/accounts.ts:471]: This one.

File: src/store/sagas/accounts.ts [hunk 2]
```diff
@@ -26,7 +26,6 @@ import {
   accountsSynched,
   settedDonationAccount,
```
Comment (AliMeer) [inline src/store/sagas/accounts.ts:27]: This one.

File: src/store/reducers/accounts.ts [hunk 3]
```diff
@@ -59,9 +59,9 @@ const ACCOUNT_VARS: {
   };
   averageTxFees: any;
   donationAccount: {
-    settedup: Boolean,
-    loading: Boolean
-  }
+    settedup: Boolean;
+    loading: Boolean;
```
Comment (AliMeer) [inline src/store/reducers/accounts.ts:63]: and these.

File: src/store/reducers/accounts.ts [hunk 4]
```diff
@@ -477,9 +477,9 @@ export default (state = initialState, action) => {
             ...state[account].donationAccount,
             settedup: action.payload.successful,
```
Comment (AliMeer) [inline src/store/reducers/accounts.ts:478]: and here

**Review by AliMeer — APPROVED** [review]

**Comment by Parsh** [comment]
Connected to #1994

**Pull request merged**